reactioncommerce/reaction
# Fix for Translation Issues in Dashboard #2672

Resolves #2672

This was also on the master branch. I noticed a [file](https://github.com/reactioncommerce/reaction/blob/master/imports/plugins/core/i18n/server/i18n/index.js) for importing translations was mostly commented hence the other languages were not being loaded to the db during startup. Others were mostly due to typos.

**Test By:**
1. Log in as Admin
2. Click on the `Localization and i18n` icon on the icon toolbar
3. While the action view part of the dashboard remains open change the language to Spanish or German
4. Observe that the translations are updating and working properly
5. Change language back to English, close the dashboard and Repeat 2 - 5 for `Payment`, and `Catalog` icons
